package.json

```diff
@@ -1411,6 +1412,11 @@
           "when": "view == confluent-flink-statements && confluent.ccloudConnectionAvailable && confluent.flinkStatementsPoolSelected",
           "group": "navigation@2"
         },
+        {
+          "command": "confluent.statements.viewResults",
+          "when": "view == confluent-flink-statements && viewItem =~ /.*flink-statement$/",
+          "group": "inline@1"
+        },
```

Comment:
Nit: this is now in between the `navigation@_` group for the Flink statements view -- was this change mainly to keep the command IDs in order? (Maybe in a separate branch we should put the `group` at the top for ordering purposes)